history-api/script.js

```diff
@@ -23,7 +23,7 @@ document.addEventListener("click", async (event) => {
       displayContent(json);
       // Add a new entry to the history.
       // This simulates loading a new page.
-      history.pushState(json, "", creature);
+      history.pushState(json, "", `${creature}.html`);
```

Comment:
When you say "advertised as" do you mean in the href? Do you think it is better to update the state, as here, or to update the href? (it feels better to me to update the href but not sure why).

Reply:
> When you say "advertised as" do you mean in the href?

Yes, when we hover over a link it shows the anchor's path in bottom left corner of the browser.

> it feels better to me to update the href but not sure why

In single page applications static looking links feel odd. Let's get rid of `.html` from paths.